src/chainstate/stacks/miner.rs

```diff
@@ -709,8 +740,29 @@ impl<'a> StacksMicroblockBuilder<'a> {
                             num_txs += 1;
                             txs_included.push(tx);
                         }
-                        TransactionResult::Skipped(..) | TransactionResult::ProcessingError(..) => {
+                        TransactionResult::Skipped(TransactionSkipped { error, .. })
+                        | TransactionResult::ProcessingError(TransactionError { error, .. }) => {
                             test_debug!("Exclude tx {} from microblock", tx.txid());
+                            match &error {
+                                Error::BlockTooBigError => {
+                                    // done mining -- our execution budget is exceeded.
+                                    // Make the block from the transactions we did manage to get
+                                    test_debug!("Block budget exceeded on tx {}", &tx.txid());
```

Comment:
Can this `test_debug!` statement be merged with the ones in either `if` branch, so that there's just one debug line emitted here?  Makes it easier to `grep` for this event in test output.